src/commands/registries/azure/DockerAssignAcrPullRoleStep.ts

```diff
@@ -4,7 +4,7 @@
  *--------------------------------------------------------------------------------------------*/
 
 import { Progress } from "vscode";
-import { IAppServiceWizardContext } from "vscode-azureappservice"; // These are only dev-time imports so don't need to be lazy
+import type { IAppServiceWizardContext } from "vscode-azureappservice"; // These are only dev-time imports so don't need to be lazy

```

Comment:
Now that we use `import type` maybe we do not need these "it is only a dev-time import" comments anymore--you decide.

Reply:
I guess I want to leave them, just for clarity on _why_ it's `import type`.